### DISCUSSION TITLE
Compatible with Statamic 3?

### DISCUSSION THREAD

**Discussion by mar627** [lucky-media/landtamic#26] [Q&A] [answered]
Hello, I noticed compatibility with v3 is mentioned here, but in the docs it reads:

> To install Landtamic, simply follow the Starter Kit installation instructions. **Note that you will need to be running Statamic 4.x.**

![CleanShot 2024-03-09 at 17 58 50@2x](https://github.com/lucky-media/landtamic/assets/16055165/88cb2ebb-c0ab-4ce1-8045-9999b34a4764)

Is Landtamic compatible with v3?

**Comment by lokmanm** [comment, marked as answer]
Hey @mar627,

This is not a Landtamic limitation but a Statamic one. When you install a starter kit it's assumed you have a fresh installation.

With Starter Kits there is no backward compatibility so I am afraid it won't work with Statamic 3.
